static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -261,6 +262,11 @@ const bootStandard = (): void => {
 
     newHeaderInit();
 
+    if (config.get('page.hasShowcaseMainElement')) {
+        //
```

Comment:
It looks like you were about to say something! 👻

Reply:
Yes! Then I realised that I could write some tests for the module, rather than comments here! 😜 👻 

Will remove this and add some comments and tests for fix-secondary-column 👍